ui/components/multichain/asset-picker-amount/asset-picker-modal/AssetList.tsx

```diff
@@ -76,6 +76,15 @@ export default function AssetList({
         const tokenAddress = token.address?.toLowerCase();
         const isSelected = tokenAddress === selectedToken?.toLowerCase();
         const isDisabled = isTokenDisabled?.(token) ?? false;
+
+        let tokenSymbol;
+        if (useNativeCurrencyAsPrimaryCurrency) {
+          tokenSymbol =
+            token.symbol === primaryCurrency ? primaryCurrency : token.symbol;
+        } else {
+          tokenSymbol = secondaryCurrency;
+        }
```

Comment:
We will be removing `useNativeCurrencyAsPrimaryCurrency` and instead; in a lot of components that used to display both fiat and crypto values will now default to showing crypto first.
I have also noticed a small bug where when the user chooses crypto:
<img width="368" alt="Screenshot 2024-10-01 at 10 19 06" src="https://github.com/user-attachments/assets/dcb85426-da46-4428-a2d9-fff6671885f6">
When the user chooses fiat: (we show $ Xamount ETH)
<img width="363" alt="Screenshot 2024-10-01 at 10 19 52" src="https://github.com/user-attachments/assets/5fb55e94-e44c-46af-b005-d98d7bebcb28">


Regardless; because in this component for ERC20 tokens we show fiat value first, and ERC20 tokens are not affected by the setting `useNativeCurrencyAsPrimaryCurrency`; do you think we can we also display ETH values similarly? (showing fiat value first then show crypto regardless of the value of `useNativeCurrencyAsPrimaryCurrency`

Reply:
> showing fiat value first then show crypto regardless of the value of useNativeCurrencyAsPrimaryCurrency

I think this makes sense for consistency among the asset list items. I fixed the symbol suffix bug and implemented what you described in the 2 most recent commits

Do you know why we show fiat first by default? I see that Swaps always shows ETH first - was this an intentional design decision?

This is what the token list looks like now, regardless of user preferences
![Screenshot 2024-10-01 at 11 08 02 AM](https://github.com/user-attachments/assets/dcc7d3d8-70aa-4a27-adcb-10c20ca0e8ee)

Reply:
I dnt have the context for that decision; on the main asset list we also show fiat first and the ERC20 asset list is not affected by the setting useNativeCurrencyAsPrimaryCurrency

Reply:
I see - Varada recommended changing the Send asset list to match Swaps' layout so I made that change in the last revision (added SS in the description). So now we always show the asset value first, then the fiat value